### PR TITLE
Use print_function from future

### PR DESCRIPTION
**Pull request by dralley**
In Python 3, 'print' was changed from a keyword into a builtin function. We can use this function from Python 2 if we import it from \_\_future\_\_.

### CODE REVIEW

**Comment by kozec** [comment]
This shouldn't really be needed, all those prints are meaningless and I can just remove them. Any normal output should go through logger.

**Comment by dralley** [comment]
Removed entirely or replaced with logging statements?

**Comment by dralley** [comment]
I can just update my PR to do this.  It might make sense to log at the debug level?

File: syncthing_gtk/app.py [hunk 1]
```diff
@@ -199,16 +199,16 @@ def do_command_line(self, cl):
 		else:
 			# Fallback for old GTK without option parsing
 			if "-h" in sys.argv or "--help" in sys.argv:
-				print "Usage:"
-				print "  %s [arguments]" % (sys.argv[0],)
-				print "Arguments:"
+				print("Usage:")
```
Comment (dralley) [inline syncthing_gtk/app.py]: except for maybe this.  the fallback probably isn't necessary anymore (although, idk what version CentOS 6/7 would be using, maybe it's old enough to still be necessary).

**Comment by kozec** [comment]
Just as side note, those *.exe and *_windows things are used only to build Windows release, so there's probably not much sense in changing them.

**Pull request closed without merging**